docs/v3/core/v3.0.rst

```diff
@@ -1797,6 +1781,7 @@ by time.
 Draft Changes
 --------------------------
 
+- Removed fallback data type support.
```

Comment:
Please add this into a separate section (e.g. "Provisionally Accepted") and also link the MR:

```suggestion
Changes after Provisional Acceptance
--------------------------

- Removed fallback data type support. `PR #248
  <https://github.com/zarr-developers/zarr-specs/pull/248>`_

Draft Changes
--------------------------

```